src/audio-worklet/free-queue/src/freequeue.mjs

```diff
@@ -61,11 +44,15 @@ export class FreeQueue {
     }
   }
 
-  static fromPointers(fqPointers: FreeQueuePointers) {
+  /**
+   * Helper function for creating FreeQueue from pointers.
+   * @param fqPointers
```

Comment:
What's the type of `fqPointers`?

Reply:
I also think this can be renamed to `queuePointer`.

Reply:
Yeah actually I was thinking of explaining structure of that in comments.
It's more of like a struct containing pointers to different data members and wasm module memory